src/pyclass.jl

```diff
@@ -66,8 +71,12 @@ function parse_pydef(expr)
     getter_dict = Dict{Any, Symbol}() # python_var => jl_getter_name
     setter_dict = Dict{Any, Symbol}()
     method_syms = Dict{Any, Symbol}() # see below
+    class_vars = Dict{Symbol, Any}()
     for line in lines
-        if !isa(line, LineNumberNode) && line.head != :line # need to skip those
+        if line isa LineNumberNode || line isa Symbol || line.head == :line
```

Comment:
Wouldn't a `Symbol` line be an error?

Reply:
Ah yes it would, I'll fix that!

Reply:
Ah I remember why I had that now: `Symbol` does not have a `head`, which would cause `parse_pydef` to error prior to this PR. Silently ignoring the error is worse, however, so I'll figure something out.